src/components/rows.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import { Box, Flex, Text } from '@blockstack/ui';
+import { Badge } from '@components/badge';
+import { Caption } from '@components/typography';
+import { Card } from '@components/card';
+import { toSnakeCase } from '@common/utils';
+
+interface TxSchema {
+  txid: string;
+  sender: string;
+  recipient: string;
+  type: number;
+  fees: {
```

Comment:
Better to make a separate type for Fees above, so it doesn't need to be repeated on line :44

Reply:
These can go now, see models folder